package.json

```diff
@@ -32,10 +32,8 @@
     "es2015/text.js"
   ],
   "scripts": {
-    "build": "babel src -d .; NODE_ENV=es2015 babel src -d ./es2015; node script/inject-sequences.js",
-    "clean": "rm -rf -- index.js text.js es2015",
-    "prepublish": "npm run clean && npm run build",
-    "test": "npm run build && mocha",
+    "build-patterns": "babel src -d .; NODE_ENV=es2015 babel src -d ./es2015; node script/inject-sequences.js",
```

Comment:
I’m used to typing `npm run build` since all my other repos use the same name, so I’d rather not change it. I’m fine with rebuilding on `npm install`.